data/tools/lozad.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name" : "lozad.js",
+  "javascript" : {
+    "url" : "https://github.com/ApoorvSaxena/lozad.js"
+  },
+  "description" : "Performant lazy loader using IntersectionObserver API",
+  "tags" : [ "loading", "lazy loading", "images", "iframes" ]
+}
```

Comment:
Hmm. Why the `iframes` tag?

Reply:
it allows lazy loading for all sort of elements be images, iframes, videos etc. and considering iframes is a very common use case thus have added in the tag

Reply:
Ja, that's what I thought. `images` works for me as a common use case, but I wouldn't add a new for `iframes`.

I'm happy to take the PR without this fourth tag. Does this work for you?